validphys2/src/validphys/config.py

```diff
@@ -1359,45 +1367,71 @@ def load_default_default_filter_settings(self, spec):
     def parse_filter_defaults(self, filter_defaults: (dict, type(None))):
         """A mapping containing the default kinematic limits to be used when
         filtering data (when using internal cuts).
-        Currently these limits are ``q2min`` and ``w2min``.
+        Currently these limits are ``q2min``, ``w2min``, and ``maxTau``.
+
+        Parameters
+        ----------
+        filter_defaults: dict, None
+            A mapping containing the default kinematic limits to be used when
+            filtering data (when using internal cuts).
+            Currently these limits are ``q2min``, ``w2min``, and ``maxTau``.
+        
+        Returns
+        -------
+        FilterDefaults
+            A hashable object containing the default kinematic limits to be used when
+            filtering data (when using internal cuts).
+            Currently these limits are ``q2min``, ``w2min``, and ``maxTau``.
         """
         log.warning("Overwriting filter defaults")
-        return filter_defaults
+
+        parsed_filter_defaults = FilterDefaults(
+                                            q2min = filter_defaults["q2min"] if "q2min" in filter_defaults else None, 
+                                            w2min = filter_defaults["w2min"] if "w2min" in filter_defaults else None,
+                                            maxTau = filter_defaults["maxTau"] if "maxTau" in filter_defaults else None,
+                                            )
```

Comment:
```suggestion
        parsed_filter_defaults = FilterDefaults(**filter_defaults)
```

And then you can initialise the attributes of the class `FilterDefaults` to `None`.

If for some reason that cannot be done, you can always do `filter_defaults.get("q2min")` which is equivalent to doing `filter_defaults["q2min"] if "q2min" in filter_defaults else None`